academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md

```diff
@@ -171,37 +197,288 @@ The input entities file Url prefix.
 
 The output url prefix for writing the built index.
 
-### Optional Parameters
+### BuildIndex optional parameters
+
+`--SchemaUrl`
+
+The url to the input index schema definition file. By default, the url is set to the schema describing the latest MAKES index release.
+
+`--SynonymResourceFolderUrl`
+
+The url to a resource folder containing all synonym data files required by the input index schema. Defaults to null.
```

Comment:
>Defaults to null. [](start = 99, length = 17)

"If not specified KESM will not attempt to download/build synonym files."

Reply:
We do not have this capability to automatically modify schema file and  exclude synonyms. 

---
In reply to: [465228691](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/172#discussion_r465228691) [](ancestors = 465228691)